lib/ansible/module_utils/network/vyos/utils/utils.py

```diff
@@ -66,3 +64,34 @@ def diff_list_of_dicts(want, have):
         diff.append(dict((x, y) for x, y in element))
 
     return diff
+
+
+def get_lst_diff_for_dicts(want, have, lst):
```

Comment:
Could you add a brief description of what these methods do?

Reply:
Done